html-relative-links/about-me/about-me.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>About Me</title>
+  <style>
+    img {
+      width: 300px;
+    }
+  </style>
+</head>
+<body>
+  <ul>
+    <li><a href='../index.html'>Home Page</a></li>
+    <li><a href='about-me.html'>About Me</a></li>
+    <li><a href='../projects/projects.html'>Projects</a></li>
+    <li><a href='../contact-us/contact-us.html'>Contact Us</a></li>
+  </ul>
+  <p>Hello About Me!</p>
+  <img src="https://anime-girls-holding-programming-books.netlify.app/static/a35ece30fa281cf15739f9464338b7fc/dd8b2/Arikawa_Hime_Holding_C_Programming_Language.png"
```

Comment:
Please use a relative path to the images included in the starter files for this exercise.